.github/workflows/tokens-diff.yml

```diff
@@ -0,0 +1,108 @@
+name: Check for token changes from css-library
+
+on:
+  pull_request:
+    types: [opened]
+
+defaults:
+  run:
+    working-directory: packages/tokens
+
+env:
+  YARN_ENABLE_IMMUTABLE_INSTALLS: false # Allows CI to modify yarn.lock
+
+jobs:
+  compare-tokens:
+    if: contains(github.event.pull_request.head.ref, 'dependabot/npm_and_yarn/packages/tokens/department-of-veterans-affairs/css-library')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          fetch-depth: 0
+          token: ${{ secrets.VA_MOBILE_ROBOT_GITHUB_PAT }}
+      - name: Set git credentials and regenerate yarn.lock
+        run: |
+          git config --global user.name 'VA Automation Bot'
+          git config --global user.email 'va-mobileapp@adhocteam.us'
+          yarn install
+
+          # Commits yarn.lock if yarn install resulted in changes
+          if git status --porcelain | grep -q '^[ MARC].*yarn.lock'; then
+            echo "yarn.lock has unstaged changes."
+            cd ../..
+            git add yarn.lock
+            git commit -m 'Regenerate yarn.lock'
+            git push
+          fi
```

Comment:
Don't feel strongly, but perhaps a little cleaner to have as two separate steps like the publish flow.